RELEASES.md

```diff
@@ -635,3 +635,9 @@
 10.2.1 2019-08-16
 -----------------
   - Issue: access denied security logs username and password via JSON.stringify
+
+10.2.2 2019-08-16
```

Comment:
this date is wrong

Reply:
thinking about removing dates and reformatting to the same way SMC does the RELEASES.md, much nicer.